app/models/region/RegionCompletionTable.scala

```diff
@@ -87,33 +83,29 @@ object RegionCompletionTable {
     * @return
     */
   def updateAuditedDistance(streetEdgeId: Int) = db.withTransaction { implicit session =>
-    val distToAdd: Float = streetEdgesWithoutDeleted.filter(_.streetEdgeId === streetEdgeId).groupBy(x => x).map(_._1.geom.transform(26918).length).list.head
-    val regionId: Int = streetEdgeNeighborhood.filter(_.streetEdgeId === streetEdgeId).groupBy(x => x).map(_._1.regionId).list.head
 
-    val q = for { regionCompletion <- regionCompletions if regionCompletion.regionId === regionId } yield regionCompletion
+    val distToAdd: Float = StreetEdgeTable.getStreetEdgeDistance(streetEdgeId)
+    val regionIds: List[Int] = streetEdgeNeighborhood.filter(_.streetEdgeId === streetEdgeId).groupBy(x => x).map(_._1.regionId).list
+
+    for (regionId <- regionIds) yield {
+      val q = for {regionCompletion <- regionCompletions if regionCompletion.regionId === regionId} yield regionCompletion
 
-    val updatedDist = q.firstOption match {
-      case Some(rC) => q.map(_.auditedDistance).update(rC.auditedDistance + distToAdd)
-      case None => -1
+      val updatedDist = q.firstOption match {
+        case Some(rC) => q.map(_.auditedDistance).update(rC.auditedDistance + distToAdd)
+        case None => -1
+      }
     }
```

Comment:
Before, the code was only updating the completion distance for a single region, even if a completed audit task intersected multiple regions. So I just modified it to update all of them.